middle_end/flambda2/from_lambda/closure_conversion.ml

```diff
@@ -154,15 +154,18 @@ let close_const0 acc (const : Lambda.structured_constant) =
   let acc, const, name = declare_const acc const in
   match const with
   | Tagged_immediate i ->
-    acc, Simple.const (Reg_width_const.tagged_immediate i), name
+    ( acc,
+      Simple.const (Reg_width_const.tagged_immediate i),
+      name,
+      Flambda_kind.With_subkind.tagged_immediate )
   | Symbol s ->
     let acc, simple = use_of_symbol_as_simple acc s in
-    acc, simple, name
+    acc, simple, name, Flambda_kind.With_subkind.any_value
```

Comment:
We don't have the ability to handle non-value symbols for now. Changing this would have non-trivial impacts (in particular in the case where a cmx file is missing...), so I think it's safe to keep assuming this property.